lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -87,14 +87,31 @@ function LoggedInUser({ userInformation, setErrorState, zendeskApiHost, submitFo
         </div>
       </div>
     </div>
-
+    showDisplay ? (
+    <div className="row">
+      <div className="col-sm-12">
+        <div className="form-group">
+          <label htmlFor="message">{gettext('Details')}</label>
+          <p
+            className="message-desc"
+          >{gettext('the more quickly and helpfully we can respond!')}</p>
+          <textarea
+            aria-describedby="message"
+            className="form-control"
+            rows="7"
+            id="message"
+          />
+        </div>
+      </div>
+    </div>
+    ) : (
     <div className="row">
       <div className="col-sm-12">
         <div className="form-group">
           <label htmlFor="message">{gettext('Details')}</label>
           <p
             className="message-desc"
-          >{gettext('The more you tell us, the more quickly and helpfully we can respond!')}</p>
+          >{gettext('Hahahhahahahha')}</p>
```

Comment:
You can add the conditional only at this stage, because the rest of the HTML is being repeated